src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
```

Comment:
Class names should not be verbs. Although 'Rectangle' is not a verb, make sure that all your class names represent entities or concepts rather than actions.